features/facebookImport/src/components/importExplanationExpandable/importExplanationExpandable.jsx

```diff
@@ -48,6 +48,18 @@ const ImportExplanationExpandable = ({
         setStartRequest(true);
     };
 
+    const exampleDataUrls = {
+        en: "https://polypoly.coop/en-de/polypod/example-facebook-data",
+        de: "https://polypoly.coop/de-de/polypod/example-facebook-data",
+    };
```

Comment:
That's not needed, you can just use the link's key from the manifest, e.g. `example-data-download-en` - though you should remove the language suffix, see my next comment.

Reply:
Done!